src/applications/facility-locator/components/ProviderServiceDescription.jsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
 import { facilityTypes, ccUrgentCareLabels } from '../config';
+import get from 'platform/utilities/data/get';
 
 export const urgentCareProviderNames = posCodes => {
   if (posCodes && parseInt(posCodes, 10) === 17) {
     return ccUrgentCareLabels.WalkIn;
   } else if (posCodes && parseInt(posCodes, 10) === 20) {
     return ccUrgentCareLabels.UrgentCare;
   } else {
-    return facilityTypes.cc_provider.toUpperCase();
+    return facilityTypes.provider.toUpperCase();
   }
 };
 
 const providerName = (query, posCodes) => {
   let name;
   switch (query.facilityType) {
-    case 'cc_pharmacy':
-      name = facilityTypes.cc_pharmacy.toUpperCase();
+    case 'pharmacy':
```

Comment:
Let's make sure this does not break the current implementation / pharmacies currently disabled on prod